folding/base/validator.py

```diff
@@ -150,6 +169,7 @@ def run(self):
                 self.sync()
 
                 self.step += 1
+                time.sleep(self.config.wait_time)
```

Comment:
So I understand the config wait time now, since the `run` method is encasing all the logic in a while loop. Therefore, we still need to update the config, I guess this was not added in push. How long do you expect the wait to be?

Reply:
Answered this in my last comment.
Wait time is the interval between querying a given job. Update interval may be a better name (and I think that’s what I called the attribute in the Job dataclass)

timeout is much shorter than wait time

Reply:
added